src/server/api/errors.js

```diff
@@ -30,6 +30,21 @@ export async function accessRequired(
   }
 }
 
+export async function assignmentOrAdminRequired(
+  user,
+  orgId,
+  assignmentId,
+  assignment
+) {
+  authRequired(user);
+  const isAdmin = await cacheableData.user.userHasRole(user, orgId, "ADMIN");
+  if (isAdmin || user.is_superAdmin) {
```

Comment:
this needs to be `user.is_superadmin` (it's a db field, and isn't case-changed)

Alternatively, you can delete it and just test on isAdmin, since the function below also tests superadmin.